warehouse/templates/base.html

```diff
@@ -210,9 +210,9 @@ <h3>Get help</h3>
             <h3>About PyPI</h3>
           </li>
           {% if request.registry.settings.get("statuspage.url") %}
-          <li><a href="https://status.python.org/">Status: <span data-statuspage-domain="{{ request.registry.settings['statuspage.url'] }}">All Systems Operational</span></a></li>
+          <li><a href="https://status.python.org/">Status: <span data-statuspage-domain="{{ request.registry.settings['statuspage.url'] }}">all systems operational</span></a></li>
           {% endif %}
-          <li><a href="https://www.python.org/dev/peps/pep-0541/">Package Index Name Retention</a></li>
+          <li><a href="https://www.python.org/dev/peps/pep-0541/">Package index name retention</a></li>
```

Comment:
No changes needed here?

Reply:
I'm less certain about this. One could argue it's a proper noun, the name of the PEP (or is the name "PEP 541"?). Or is it just the title of the PEP and they follow a different capitalisation style over there?

https://www.python.org/dev/peps/pep-0541/

Reply:
I think this change is fine.